Formula/stern.rb

```diff
@@ -0,0 +1,32 @@
+class Stern < Formula
+  desc "Tail multiple Kubernetes pods & their containers."
+  homepage "https://github.com/wercker/stern"
+  url "https://github.com/wercker/stern.git",
+    :tag => "1.0.1",
+    :revision => "3c8e6184b7a0a0cbc8dc78f219be0a49a007e861"
```

Comment:
This doesn't seem to use submodules so we prefer https://github.com/wercker/stern/archive/1.0.1.tar.gz